### ISSUE TITLE
Tap and hold to open the bookmarks/history menu is awful

### ISSUE THREAD

**Issue by KurobaExReports** [KurobaExReports/Reports#674]
Make it like it KurobaEx, dragging from left to the right in the middle of the screen = back to catalog, dragging from the edge to the right = bookmarks/history. Let Android handle the tap and hold thing, my device for example has an exclusion zone on the top left of the screen, but if I wanted I could tap and hold anywhere too (for what feels like a millisecond, not a whole second like in this app).

**Comment by K1rakishou** [comment]
If you disable history (or move it to the right side) you will be able to drag the drawer from any point of the screen (when in catalog mode).

**Comment by K1rakishou** [comment]
Also this

>dragging from left to the right in the middle of the screen = back to catalog, dragging from the edge to the right = bookmarks/history.

is currently not possible to do in jetpack compose correctly. It will start blocking regular taps. That's why I removed that special zone. I will probably bring it back when they implement the necessary APIs to do that correctly.

**Comment by K1rakishou** [comment]
>my device for example has an exclusion zone on the top left of the screen, but if I wanted I could tap and hold anywhere too (for what feels like a millisecond, not a whole second like in this app).

Also this is something very custom and does not exist in regular Android (AOSP) so I can't do anything with it.

**Comment by KurobaExReports** [comment]
Yeah I know it works from catalog and that exclusions don't exist in AOSP. But the native "tap and hold then drag to open navigation drawer" is still faster than the one KurobaEx Lite uses.

**Comment by K1rakishou** [comment]
Yeah, for some weird reason it might actually be slower than the native one.
